<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equator - LaTeX Snippet Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonErrorLog">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Error Log</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/dialog-error.svg</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCopyClipboard">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Copy to Clipboard</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/edit-redo.svg</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCompile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Com&amp;pile</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/go-next.svg</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Export">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionFileExportSvg"/>
     <addaction name="actionFileExportPng"/>
    </widget>
    <addaction name="menu_Export"/>
    <addaction name="actionFileImport"/>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditCopy"/>
    <addaction name="actionEditPaste"/>
    <addaction name="actionEditOptions"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewZoomIn"/>
    <addaction name="actionViewZoomOut"/>
    <addaction name="actionViewNormalSize"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionViewZoomIn"/>
   <addaction name="actionViewZoomOut"/>
   <addaction name="actionViewNormalSize"/>
  </widget>
  <action name="actionFileQuit">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/edit-logout.svg</normaloff>:/icons/edit-logout.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEditOptions">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/preferences-system.svg</normaloff>:/icons/preferences-system.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionButtonCompile">
   <property name="text">
    <string>actionButtonCompile</string>
   </property>
  </action>
  <action name="actionButtonErrorLog">
   <property name="text">
    <string>actionButtonErrorLog</string>
   </property>
  </action>
  <action name="actionButtonCopyClipboard">
   <property name="text">
    <string>actionButtonCopyClipboard</string>
   </property>
  </action>
  <action name="actionViewZoomIn">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/zoom-in</normaloff>:/icons/zoom-in</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionViewZoomOut">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/zoom-out</normaloff>:/icons/zoom-out</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionViewNormalSize">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/zoom-original</normaloff>:/icons/zoom-original</iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileExportSvg">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/export-document</normaloff>:/icons/export-document</iconset>
   </property>
   <property name="text">
    <string>SVG</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileExportPng">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/export-document</normaloff>:/icons/export-document</iconset>
   </property>
   <property name="text">
    <string>PNG</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileImport">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/import-document.svg</normaloff>:/icons/import-document.svg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/about.svg</normaloff>:/icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/edit-copy.svg</normaloff>:/icons/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy to Clipboard</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="icon">
    <iconset resource="equagen.qrc">
     <normaloff>:/icons/edit-paste.svg</normaloff>:/icons/edit-paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste from Clipboard</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="equagen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCompile</sender>
   <signal>clicked()</signal>
   <receiver>actionButtonCompile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonErrorLog</sender>
   <signal>clicked()</signal>
   <receiver>actionButtonErrorLog</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCopyClipboard</sender>
   <signal>clicked()</signal>
   <receiver>actionButtonCopyClipboard</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
